src/powershellScripts.ts

```diff
@@ -94,16 +105,34 @@ $testSuiteInfo = [PSCustomObject]@{
     children = [Collections.Generic.List[Object]]@()
 }
 
-foreach ($file in $found) {
-    $fileSuite = [PSCustomObject]@{
-        type = 'suite'
-        id = $file.BlockContainer.Item.FullName
-        file = $file.BlockContainer.Item.FullName
-        label = $file.BlockContainer.Item.Name
-        children = [Collections.Generic.List[Object]]@()
+if (($PesterVersion.Major -eq 5 -and $PesterVersion.Minor -ge 2) -or
```

Comment:
Might make more sense to move this up above, or put the old and new logic into separate functions, so that you aren't repeating the version check logic twice (and thus having to remember to update it in two places if it changes)